pkg/securitypolicy/regopolicy_test.go

```diff
@@ -178,7 +178,7 @@ func Test_Rego_EnforceOverlayMountPolicy_No_Matches(t *testing.T) {
 			return false
 		}
 
-		err = tc.policy.EnforceOverlayMountPolicy(tc.containerID, tc.layers, testDataGenerator.uniqueMountTarget())
+		err = tc.policy.EnforceOverlayMountPolicy(tc.containerID, tc.layers, testDataGenerator.uniqueMountTarget(), "", "")
```

Comment:
Assuming that you will be adding proper generative test values to these functions.

Reply:
ditto